tests/conftest.py

```diff
@@ -240,26 +241,11 @@ def _config_sbatch_supervisor_env(env):
 
 
 def _job_supervisor_check(env):
-    import socket
-
-    s = socket.socket(socket.AF_INET, socket.SOCK_STREAM)
-    s.setsockopt(socket.SOL_SOCKET, socket.SO_REUSEADDR, 1)
-    try:
-        s.bind(
-            (
-                env.SIREPO_PKCLI_JOB_SUPERVISOR_IP,
-                int(env.SIREPO_PKCLI_JOB_SUPERVISOR_PORT),
-            )
-        )
-    except Exception:
-        raise AssertionError(
-            "job_supervisor still running on ip={} port={}".format(
-                env.SIREPO_PKCLI_JOB_SUPERVISOR_IP,
-                env.SIREPO_PKCLI_JOB_SUPERVISOR_PORT,
-            ),
-        )
-    finally:
-        s.close()
+    _ = _port(
```

Comment:
The assignment to _ is unnecessary and confusing.

Reply:
[a373bd2](https://github.com/radiasoft/sirepo/pull/4846/commits/a373bd279a8441720bda7dbf8a71bb3fe53e54b2)